### PR TITLE
Better emoji plugin documentation

### PR DESCRIPTION
**Pull request by andrrms**
This includes better documentation and keeps up with changes proposed [here](https://github.com/grammyjs/emoji/pull/9)

### CODE REVIEW

File: site/docs/plugins/emoji.md [hunk 1]
```diff
@@ -73,12 +76,20 @@ bot.use(emojiParser());
 </CodeGroupItem>
 </CodeGroup>
 
-And you can get emojis in your replies like this:
+Now, whenever you want, you can get an emoji by name like this:
```
Comment (roj1512) [inline site/docs/plugins/emoji.md:79]:
```suggestion
Now you can get emojis by names like this:
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
The changes at https://github.com/grammyjs/emoji/pull/9 are unlikely to be merged. This PQ needs to be reduced to the improvements unrelated to the aforementioned PQ.

**Comment by roj1512** [comment]
@andrrms I think it might be better to open documentation update PRs only when the current one is outdated (=after you have already made an update to your plugin), or maybe just mark it as a draft until the update on the other side (the side the documentation talks about, which is your plugin in this context) is complete.

**Review by roj1512 — APPROVED** [review]

**Comment by roj1512** [comment]
@andrrms Good move.

**Comment by KnorpelSenf** [comment]
I think we need to communicate to everyone that they are not supposed to merge their own PQs. This happened repeatedly by different people now, so it looks like no one knows

**Comment by roj1512** [comment]
Should I tag everyone in the chat and tell them?

**Comment by WingLim** [comment]
> > @andrrms ~Machiavélique~ move.
> 
> I saw a big shiny green button and no warning that I shouldn't merge yet, sorry about that 😅

It doesn't matter, you did a great job already.

**Pull request merged**